CMakeLists.txt

```diff
@@ -1,161 +1,74 @@
 # CMAKE project for openrct2
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.9)
 if (CMAKE_BINARY_DIR STREQUAL CMAKE_SOURCE_DIR)
     message(FATAL_ERROR "Building in-source is not supported! Create a build dir and remove ${CMAKE_SOURCE_DIR}/CMakeCache.txt")
 endif()
 
-project(openrct2)
+project(openrct2 VERSION 0.1.5 LANGUAGES C CXX)
```

Comment:
Is this version needed? Where is it used? Do we have to take care of it when doing releases?